os_stub/spdm_crypt_ext_lib/cryptlib_ext.h

```diff
@@ -491,6 +491,23 @@ extern bool libspdm_ec_check_key(const void *ec_context);
 extern bool libspdm_ecd_set_pub_key(void *ecd_context, const uint8_t *public_key,
                                     size_t public_key_size);
 
+/**
+ * Sets the private key component into the established Ed context.
+ *
+ * For ed25519, the private_size is 32.
+ * For ed448, the private_size is 57.
+ *
+ * @param[in, out]  ecd_context      Pointer to Ed context being set.
+ * @param[in]       private         Pointer to the buffer to receive generated private X,Y.
+ * @param[in]       private_size     The size of private buffer in bytes.
+ *
+ * @retval  true   Ed private key component was set successfully.
+ * @retval  false  Invalid EC private key component.
+ *
+ **/
+bool libspdm_ecd_set_pri_key(void *ecd_context, const uint8_t *private_key,
```

Comment:
This should be in `hal/library/cryptlib` since it's needed by one of the core libraries.

Reply:
Thanks. 
I have move `libspdm_ecd_set_pub_key`/`libspdm_ecd_set_pri_key`/`libspdm_ecd_get_pub_key` declaration to `hal/library/cryptlib/cryptlib_ecd.h`, because the `library/spdm_crypt_lib/fips` use these three fucntions.

And I think such as `libspdm_ec_set_priv_key` declaration should move to `hal/library/cryptlib`. I will submit issue and PR to fix it.
https://github.com/DMTF/libspdm/blob/51c1cf4ecde9a9b2c7996217c86bebc97a212917/os_stub/spdm_crypt_ext_lib/cryptlib_ext.h#L429-L445

Reply:
About other function, I have submit PR: https://github.com/DMTF/libspdm/pull/1845 to fix.